dm/worker/server.go

```diff
@@ -177,7 +185,12 @@ func (s *Server) OperateSubTask(ctx context.Context, req *pb.OperateSubTaskReque
 	var err error
 	switch req.Op {
 	case pb.TaskOp_Stop:
-		err = s.worker.StopSubTask(name)
+		if err = s.worker.meta.Del(name); err != nil {
+			log.Errorf("update task %s into meta: %v", name, errors.ErrorStack(err))
+			resp.Msg = fmt.Sprintf("update task %s into meta: %v", name, errors.ErrorStack(err))
```

Comment:
you can reuse this message both for `log.Errorf` and `resp.Msg`.

Reply:
LOL, in fact I want to change it